### DISCUSSION TITLE
Very basic PDF is huge

### DISCUSSION THREAD

**Discussion by hectorgimenez** [gotenberg/gotenberg#627] [Q&A]
Hello! First of all thanks for building this tool!

I'm moving from wkhtmltopdf to gotenberg, I'm just using the HTML to PDF conversion and tried with one HTML (about 220kb, no images or external resources, embedded css). Using wkhtmltopdf I get a 160KB PDF, with gotenberg it's 1.95MB, 1.42MB using PDF/A-1a format. Not a big problem for my use case, but surprising to me the huge difference. I've been reading this issue https://github.com/gotenberg/gotenberg/issues/521 but I'm not using any custom font. Also tried the omitBackground param but same result.

One thing that I noticed is that opening the PDF with a super old computer I can see how text is getting rendered "slow" (maybe 1-2 seconds later than the tables), this is not happening with the 160KB PDF.

Is this expected? Honestly not sure how can I debug a bit more on this topic, or how to provide more relevant information.

**Comment by gulien** [comment]
Hello @hectorgimenez,

I'm not sure I can help much more than what I've been said in #521. What is the font family you're using?

**Comment by hectorgimenez** [comment]
I tried with Arial and also removing the font on css (which I suppose will use the chromium default one). But same result.

**Reply by gulien** [reply]
I guess it's a Chrome issue alas 🤔

**Reply by gulien** [reply]
All issues seem to be related to image size (for instance https://bugs.chromium.org/p/chromium/issues/detail?id=414976), but you have none in your PDF. Or is Chrome generating some?

**Reply by hectorgimenez** [reply]
Nope, very simple html, no images. But it has many tables and rows. I think you are right and it's a chrome issue, I tried to open the generated HTML and used the "save as PDF" functionality and it generated a 2MB pdf size, so not a gotenberg issue for sure. Probably something wrong on the html or just that chrome don't like my tables.
